controllers/seedgen_controller.go

```diff
@@ -597,18 +596,14 @@ func (r *SeedGeneratorReconciler) restoreSeedgenCRIfNeeded(ctx context.Context,
 	// Save status as the seedgen structure gets over-written by the create call
 	// with the result which has no status
 	status := seedgen.Status
-	if err := common.RetryOnConflictOrRetriable(retry.DefaultBackoff, func() error {
-		return client.IgnoreAlreadyExists(r.Client.Create(ctx, seedgen)) //nolint:wrapcheck
```

Comment:
Is it intended to remove the alreadyExists check?

Reply:
Nope! Deleted too much. Brought it back!